### PR TITLE
Finalisation CSS Second layer

### PR DESCRIPTION
**Pull request by AdrianBor**
Could you check and integrate that part ? for the bottom it’s the most close version for the moment. Tel me about the structure.

### CODE REVIEW

**Comment by lukaswelte** [comment]
@AdrianBor: Could you add a picture (here in the comments) previewing the second layer page like it is with the changes?

File: src/components/EventDetail.js [hunk 1]
```diff
@@ -49,13 +49,19 @@ class EventDetail extends Component {
...
         <View style={styles.stepContainer}>
           <View style={styles.labelRow}>
-            <Text style={{marginHorizontal: 5}}>Sun</Text>
-            <Text style={{marginHorizontal: 5}}>Multi</Text>
-            <Text style={{marginHorizontal: 5}}>Nice</Text>
+            <Text style={styles.eventInfos}>Sun</Text>
+            <Text style={styles.eventInfos}>Multi</Text>
+            <Text style={styles.eventInfos}>Nice</Text>
+          </View>
+          <View style={styles.stepStatus}>
```
Comment (lukaswelte) [inline src/components/EventDetail.js:56]: What is this doing? It is an empty view?

File: src/components/EventDetail.js [hunk 2]
```diff
@@ -134,19 +140,37 @@ const styles = StyleSheet.create({
...
+        width: 28,
+        height: 28,
+        marginHorizontal: 10,
+        justifyContent: 'center',
+        backgroundColor: 'black',
+        color: 'white',
+        textAlign: 'center',
+      },
+  stepStatus: {
+    flex: 0,
```
Comment (lukaswelte) [inline src/components/EventDetail.js:160]: When the height is set static, do you really need `flex: 0` or can't it be removed?

**Comment by AdrianBor** [comment]
ios v4 - second layer gabarit_half
ios v4 - second layer gabarit_modifs
ios v4 - second layer gabarit_old

**Comment by lukaswelte** [comment]
Great thanks. I'm keeping this for later as I'm currently integrating the steps in the screen and will merge your design changes afterwards.

**Comment by AdrianBor** [comment]
Ok I will... do something else :)

**Pull request closed without merging**